commands2/button/networkbutton.py

```diff
@@ -0,0 +1,52 @@
+from networktables import NetworkTable, NetworkTables, NetworkTableEntry
+
+from typing import Union, overload
+
+from .button import Button
+
+
+class NetworkButton(Button):
+    """
+    A class used to bind command scheduling to a NetworkTable boolean fields.
+    Can be composed with other buttons with the operators in Trigger.
+
+    @see Trigger
+    """
+
+    @overload
+    def __init__(self, entry: NetworkTableEntry) -> None:
+        """
+        Creates a NetworkButton that commands can be bound to.
+
+        :param entry: The entry that is the value.
+        """
+
+    @overload
+    def __init__(self, table: Union[NetworkTable, str], field: str) -> None:
+        """
+        Creates a NetworkButton that commands can be bound to.
+
+        :param table: The table where the networktable value is located.
+        :param field: The field that is the value.
+        """
+
+    def __init__(self, *args, **kwargs) -> None:
+        num_args = len(args) + len(kwargs)
+        if num_args == 1:
+            entry: NetworkTableEntry = kwargs.get("entry", args[0])
```

Comment:
Won't the `args[0]` mean this cannot be passed in as a kwarg?

Reply:
No, it tries `kwargs["entry"]` first then falls back to `args[0]`

Reply:
Isn't the order of operations here to fetch `args[0]` then call `kwargs.get`?

Reply:
Oh that's right. I guess
```suggestion
            entry: NetworkTableEntry = kwargs.get("entry") or args[0]
```
would do then.